shortuuid.go

```diff
@@ -73,7 +73,11 @@ func (su *ShortUUID) Encode(u uuid.UUID) string {
 // Decode decodes a string according to the alphabet into a uuid.UUID. If s is
 // too short, its most significant bits (MSB) will be padded with 0 (zero).
 func (su *ShortUUID) Decode(u string) (uuid.UUID, error) {
-	return uuid.FromString(su.stringToNum(u))
+	str, err := su.stringToNum(u)
+	if err != nil {
+		return uuid.Nil, err //TODO: the first return is questionable
```

Comment:
I think this is okay, an error has occurred after all and the returned uuid shouldn't be used anyway.